sequencer/src/persistence/fs.rs

```diff
@@ -509,7 +509,7 @@ impl Inner {
         if self.legacy_anchor_leaf_path().is_file() {
             // We may have an old version of storage, where there is just a single file for the
             // anchor leaf. Read it and return the contents.
-            let mut file = File::open(self.legacy_anchor_leaf_path())?;
+            let mut file = BufReader::new(File::open(self.legacy_anchor_leaf_path())?);
```

Comment:
@imabdulbasit FYI, this is https://rust-lang.github.io/rust-clippy/master/index.html#unbuffered_bytes